### PR TITLE
fix name of Browning model; closes #841

### LINKED ISSUE #841: Incorrect citation information for Browning American Admixture model
The demographic model for American admixture from Browning et al has the id "AmericanAdmixture_4B11" and the year is given as 2011. But this is a 2018 paper that is based on the Gravel et al 2011 model (which is maybe where the year confusion came from).

### PR DESCRIPTION
**Pull request by petrelharp**
See discussion in #841; changes the name and provides an alias (and throws a deprecation warning).

### CODE REVIEW

**Comment by petrelharp** [comment]
This is on top of #1602.

**Comment by petrelharp** [comment]
"Oh, this will be easy!"
... half an hour later...

**Review by nspope — COMMENTED** [review]
LGTM -- put a blurb in the changelog, too?

File: docs/tutorial.rst [hunk 1]
```diff
@@ -1470,8 +1470,8 @@ This prints detailed information about all of the available models to
 the terminal.
 In this tutorial, we will use the model of African-American admixture from
 `2011 Browning et al <http://dx.doi.org/10.1371/journal.pgen.1007385>`_.
```
Comment (nspope) [inline docs/tutorial.rst:1472]:
```suggestion
`2018 Browning et al <http://dx.doi.org/10.1371/journal.pgen.1007385>`_.
```

Comment (nspope) [reply docs/tutorial.rst:1472]: I see this is done in #1604, actually

**Pull request merged**